extensions/amp-recaptcha-input/0.1/amp-recaptcha-input.js

```diff
@@ -80,6 +78,8 @@ export class AmpRecaptchaInput extends AMP.BaseElement {
 
     return this.mutateElement(() => {
       toggle(this.element);
+      // Add the required AsyncInput class
+      this.element.classList.add(AsyncInputClasses.ASYNC_INPUT);
```

Comment:
nit: Also need to use `AsyncInputClasses['ASYNC_INPUT']` to avoid the value get mangled.

Reply:
Ahhh nice catch. Thank you!